metabase/metabase
# add both option for pie chart percentages

Closes https://github.com/metabase/metabase/issues/40617

### Description

An additional radio option for the `pie.percent_visibility` has been added that will show percentages on the both the chart and legend simultaneously.

### How to verify

1. Create a pie chart
2. Set "show percentages" setting to "both"
3. Confirm percentages are visible on chart and legend.

### Demo

https://github.com/user-attachments/assets/af25d628-07a6-4031-b8b9-dff93c42bc95

### Checklist
